### PR TITLE
[quest] Felwood Quest Fixes (Cata)

### PR DESCRIPTION
**Pull request by coldpasta123**
## Issue references

Fixes #

## Proposed changes

-
-

### CODE REVIEW

File: Database/Corrections/cataObjectFixes.lua [hunk 1]
```diff
@@ -333,6 +333,11 @@ function CataObjectFixes.Load()
             [objectKeys.spawns] = {[zoneIDs.FELWOOD]={{49.03,84.95},{48.74,84.98},{48.58,84.43},{48.34,84.62},{46.92,88.14},{46.52,90.53},{45.78,86.46},{47.34,85.70},{48.19,87.39},{51.19,84.19},{50.76,85.02},{49.94,86.24},{49.74,86.48},{49.12,86.24},{48.73,86.30},{48.75,87.55},{48.83,88.62},{49.46,89.44},{48.89,89.28},{48.29,89.76},{49.08,90.81},{49.00,91.35},{48.47,92.23},{48.23,92.26},{47.74,91.68},{48.35,91.20},{48.35,91.20}}},
             [objectKeys.zoneID] = zoneIDs.FELWOOD,
         },
+        [460016] = {
+            [objectKeys.name] = "Felwood Rich Soil",
```
Comment (BreakBB) [inline Database/Corrections/cataObjectFixes.lua]: Same as before: The name needs to match.

File: Database/Corrections/cataObjectFixes.lua [hunk 2]
```diff
@@ -328,6 +328,11 @@ function CataObjectFixes.Load()
         [460014] = {
             [objectKeys.name] = "Mark of the World Tree",
         },
+        [460015] = {
+            [objectKeys.name] = "Felwood Bee Hive",
```
[the remark this answers is not in the document]
Comment (coldpasta123) [reply Database/Corrections/cataObjectFixes.lua:332]: Another visual marker that is not a 'thing'. Would it be better to set the coords as drop locations for the quest item instead? (Deadwood Honey Glob)

File: Database/Corrections/cataObjectFixes.lua (hunk 1 above)
Comment (coldpasta123) [reply Database/Corrections/cataObjectFixes.lua]: Yet another visual marker that is not a 'thing'. Would it be better to set the coords as drop locations for the quest bunny npc instead? (Irontree Soil Credit)

File: Database/Corrections/cataObjectFixes.lua (hunk 2 above)
[the remark this answers is not in the document]
Comment (BreakBB) [reply Database/Corrections/cataObjectFixes.lua:332]: For items you can't directly set any drop locations, but only through NPCs and objects. So this way is totally fine 👍🏻

File: Database/Corrections/cataObjectFixes.lua (hunk 1 above)
Comment (BreakBB) [reply Database/Corrections/cataObjectFixes.lua]: Nah, it's good to replace the bunny objectives.

File: Database/Corrections/cataQuestFixes.lua [hunk 3]
```diff
@@ -7666,28 +7710,55 @@ function CataQuestFixes.Load()
...
+        },
         [28218] = { -- A Destiny of Flame and Sorrow
+            [questKeys.preQuestSingle] = {28217},
             [questKeys.objectives] = {{{47733}}},
         },
+        [28219] = { -- Buzzers for Baby
+            [questKeys.preQuestSingle] = {28229},
+        },
+        [28220] = { -- Seeking Soil
+            [questKeys.objectives] = {nil, {{460016,nil,Questie.ICON_TYPE_EVENT}}},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:7743]:
```suggestion
            [questKeys.objectives] = {nil,{{460017,nil,Questie.ICON_TYPE_EVENT}}},
```

File: Database/Corrections/cataQuestFixes.lua [hunk 4]
```diff
@@ -7492,6 +7492,9 @@ function CataQuestFixes.Load()
         [28149] = { -- Whispers in the Wind
             [questKeys.preQuestSingle] = {28133},
         },
+        [28150] = { -- An Arcane Ally
+            [questKeys.exclusiveTo] = {28000},
```
[the remark this answers is not in the document]
Comment (BreakBB) [reply Database/Corrections/cataQuestFixes.lua:7629]:
```suggestion
            [questKeys.nextQuestInChain] = 28000,
```

File: Database/Corrections/cataQuestFixes.lua [hunk 5]
```diff
@@ -7588,6 +7624,14 @@ function CataQuestFixes.Load()
         [28149] = { -- Whispers in the Wind
             [questKeys.preQuestSingle] = {28133},
         },
+        [28150] = { -- An Arcane Ally
+            [questKeys.preQuestGroup] = {27995,27997,28148},
+            [questKeys.exclusiveTo] = {28000},
+        },
+        [28152] = { -- Jaedenar Awaits
+            [questKeys.preQuestSingle] = {28113},
+            [questKeys.exclusiveTo] = {28116},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:7633]:
```suggestion
            [questKeys.nextQuestInChain] = 28116,
```

**Pull request merged**